expirationd.lua

```diff
@@ -159,7 +159,7 @@ local function worker_loop(task)
     fiber.name(string.format("worker of %q", task.name), { truncate = true })
 
     while true do
-        if (box.cfg.replication_source == nil and box.cfg.replication == nil) or task.force then
+        if ((box.cfg.replication_source == nil and box.cfg.replication == nil) or task.force) and (not box.info.ro) then
```

Comment:
Sorry for empty pr message.

So. 

I think about expirationd in terms

expirationd has two parts
 - scheduler (while true ...)
 - expiration mutation (in particulray space:delete)

storage system can idempendtly switch state from/to readonly
state switching happends without user control

this way user runs expirationd on every nodes he want and scheduler runs on every nodes

but when storage system goes to read-only state mutation step have to be skipped

The answer is I want to disable mutation part of expirationd.

Reply:
Hm. We can set read-only for a node out of a cluster (ref. above as singleton node), but there are no much reason. It seems that you are about cluster really and want expirationd lean on read-only flag automatically: enable when writeable, disable when read-only. This can be considered as default or under-flag behaviour.

Reply:
Yes, it is about cluster.